### PR TITLE
Add service level patching for renaming service log group name

### PR DESCRIPTION
**Pull request by ejayesh**
Small change to bootstrap to include service level cfn patching to rename log group name with hypens to /'s

### CODE REVIEW

File: dbt_copilot_helper/templates/svc/overrides/cfn.patches.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- op: replace
+  path: /Resources/LogGroup/Properties/LogGroupName
+  value: !Sub '/copilot/${AppName}/${EnvName}/${WorkloadName}'
```
Comment (acodeninja) [inline dbt_copilot_helper/templates/svc/overrides/cfn.patches.yml:3]: No new line

File: dbt_copilot_helper/commands/bootstrap.py [hunk 2]
```diff
@@ -94,6 +102,7 @@ def make_config(directory="."):
         )
         name = service["name"]
         (base_path / f"copilot/{name}/addons/").mkdir(parents=True, exist_ok=True)
+        _generate_svc_overrides(base_path, templates, name)
```
Comment (acodeninja) [inline dbt_copilot_helper/commands/bootstrap.py:105]: Not something to address here, but what about completely new applications not created with a bootstrap file.

Comment (ejayesh) [reply dbt_copilot_helper/commands/bootstrap.py:105]: yep will create a new ticket to pick that up

File: dbt_copilot_helper/templates/svc/overrides/cfn.patches.yml (hunk 1 above)
Comment (ejayesh) [reply dbt_copilot_helper/templates/svc/overrides/cfn.patches.yml]: added.

**Pull request merged**